pep-0201.txt

```diff
@@ -57,63 +61,68 @@ Lockstep For-Loops
     >>> map(None, a, b)
     [(1, 4), (2, 5), (3, 6)]
 
-    The for-loop simply iterates over this list as normal.
+The for-loop simply iterates over this list as normal.
+
+While the map() idiom is a common one in Python, it has several
+disadvantages:
 
-    While the map() idiom is a common one in Python, it has several
-    disadvantages:
+* It is non-obvious to programmers without a functional programming
+  background.
 
-    - It is non-obvious to programmers without a functional
-      programming background.
+* The use of the magic ``None`` first argument is non-obvious.
 
-    - The use of the magic `None' first argument is non-obvious.
+* It has arbitrary, often unintended, and inflexible semantics when
+  the lists are not of the same length: the shorter sequences are
+  padded with ``None``.
 
-    - It has arbitrary, often unintended, and inflexible semantics
-      when the lists are not of the same length: the shorter sequences
-      are padded with `None'.
+::
```

Comment:
I think this example is part of the "It has arbitrary, often unintended, [...]" item above. I'd suggest the following markup:

```rst
* [...]
  [...]
  padded with ``None``::

      >>> c = (4, 5, 6, 7)
      # rest of the example
```